recipes/tibanna/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "tibanna" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "5839060716353649a4e482dc7e25eaec26ffdf048ad4c7695fd2991f9a473e3f"
+
+build:
+  number: 0
+  entry_points:
+    - tibanna = tibanna.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [not py3k]
+
+requirements:
+  host:
+    - python ==3.6
```

Comment:
This is not needed and will even cause a lot of problems when we move on to newer python versions. Which python versions to build for is rather defined by the `skip` statement in line 17. Does that mean that tibanna does not build for python 3.7 e.g.?

Reply:
The comment behind the skip statement is a so-called selector, which tells conda to only keep that line if the condition in the brackets is true. Here, the condition is `not py3k` meaning that the build will be skipped for python 2.x, but not for python 3.x.

Reply:
The `continuous-integration/appveyor/pr` test fails if I don't set `python ==3.6` - I think it's using python 2.7. I just wanted to specify a version (3.6) that is standard runtime for the lambdas that we deploy (we haven't tested it on 3.7), but the reason I set this was because the test was using python 2.7, which we do not support for sure.